### PR TITLE
Starting the bio for soonernate

### PR DESCRIPTION
**Pull request by soonernate**
Adding the bio file to the repo. Initial population of base biography placeholder.
## Start of a bio
- [x] get feedback from at least one team member
- [x] add favorite restaurant
- [x] add more hobbies

@loranallensmith am I missing anything?

Responding to issue #12

### CODE REVIEW

File: soonernate-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# About Nathan Aker
+
+- Lives in Edmond, OK
```
Comment (soonernate) [inline soonernate-bio.md:3]: What is this?

Comment (hydroweaver) [inline soonernate-bio.md:3]: this is good stuff !!

**Comment by pgdesign** [comment]
- Add your favorite drink

**Comment by soonernate** [comment]
:ship: Ship It

**Pull request merged**